tests/kernel/mem_protect/sys_sem/src/main.c

```diff
@@ -116,8 +116,12 @@ void sem_take_multiple_high_prio_helper(void *p1, void *p2, void *p3)
 
 void sem_multiple_threads_wait_helper(void *p1, void *p2, void *p3)
 {
+	s32_t ret_value;
```

Comment:
By the way, if you are changing things, maybe it would be better to set this as int to match the sys_sem_take return type